CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * None
 
+### Fixed
+
+* `New-TssSession` - fixed token type when AccessToken parameter is used. [26720ba](https://github.com/thycotic-ps/thycotic.secretserver/pull/73/commits/26720ba9d37b3f2c6c8986a2f1b2ed30dc5c20e7)
```

Comment:
This can be put under the changed section.

Reply:
in https://github.com/thycotic-ps/thycotic.secretserver/pull/73/commits/f8082dc462a413652de34ca53e88c8cc9f2b0d4e